install/installing-with-kata-manager.md

```diff
@@ -0,0 +1,37 @@
+# Installing with kata-manager
+
+* [Introduction](#Introduction)
+* [Full Installation](#full-installation)
+* [Install the Kata packages only](#install-the-kata-packages-only)
+* [Further Information](#further-information)
+
+## Introduction
+`kata-manager` automates the Kata Containers installation procedure documented for [these Linux distributions](README.md#supported-distributions).
+
+> Note:
+> - Full installation mode is only available for Docker container manager. For other setups, you
+> can still use `kata-manager` to [install Kata package](#install-kata-packages-only), and then setup your container manager manually.
+
+## Full Installation
+This command will:
```

Comment:
Lines 16-19 suggested rewrite: 
This command does the following: 
1. Installs Kata Containers packages
2. Installs Docker
3. Configures Docker to use the kata OCI runtime by default